supplementary_style_guide/style_guidelines/grammar.adoc

```diff
@@ -103,4 +103,28 @@ The following list includes homographs that might commonly appear in technical d
 * object
 * project
 
+[[minimalism]]
+== Minimalism
+Minimalism is a methodology for creating targeted documentation focused on your readers' needs. If you understand your customers' needs, you can write shorter and simpler documentation specific to what customers want to do. For more information about minimalism, see link:https://source.redhat.com/groups/public/minimalism-quality-initiative[Minimalism in Red Hat documentation].
+
+There are five principles of minimalism:
+
```

Comment:
I'd rewrite this to: "Minimalism has five principles:"

Reply:
Fixed